src/main/resources/core/extensions-support-overrides.json

```diff
@@ -492,7 +492,7 @@
       "artifact-id": "quarkus-reactive-routes",
       "metadata": {
         "redhat-support": [
-          "supported"
```

Comment:
@cescoffier are you sure this should be under "redhat-support"? I thought it's either supported or not (possibly as a tech-preview). `deprecated` is a tag that could be applied in the community catalog and it will still show up on code.quarkus.redhat.com next to the redhat-support tag.
@maxandersen @tqvarnst wdyt about it?

Reply:
For example https://github.com/quarkusio/quarkus/blob/main/extensions/reactive-routes/runtime/src/main/resources/META-INF/quarkus-extension.yaml
could change status to `deprecated` as it's done here
 https://github.com/quarkusio/quarkus/blob/main/extensions/resteasy-classic/rest-client-mutiny/runtime/src/main/resources/META-INF/quarkus-extension.yaml#L14

Reply:
It is supported, and now we deprecate it.

Reply:
(But it is still supported)

Reply:
I see; I've removed "supported".

Reply:
> `deprecated` is a tag that could be applied in the community catalog and it will still show up on code.quarkus.redhat.com next to the redhat-support tag.

can you specific what you refer to here - "deprecated" in "status" key is separate from "deprecated" in "redhat-supported" - I would definitely recommend we don't mix those up if can be avoided.

Reply:
note, this is most likely caused by code.quarkus limitation that "flattens" the model into one set of tags which is unfortunate :/ all other devtools are able to differentiate.